Datasets/data-connect-dataset-user1.md

```diff
@@ -70,6 +70,7 @@ The User_v1 dataset can be joined with other datasets that can be mapped to the
 | imAddresses  | array  | The instant message voice over IP (VOIP) session initiation protocol (SIP) addresses for the user. *Format:*`ARRAY<STRING>.`  | No  | None  |
 | interests  | array  | A list of interests for the user. *Format:*`ARRAY<STRING>.`  | No  | None  |
 | jobTitle  | string  | The user’s job title.  | No  | None  |
+| lastPasswordChangeDateTime  | datetime  | The time when this Microsoft Entra user last changed their password or when their password was created, whichever date the latest action was performed.  | No  | None  |
```

Comment:
1) I dont see this field in the schema https://o365exchange.visualstudio.com/O365%20Core/_git/MARS?path=/sources/dev/mars/src/DiscoveryService/Data/BasicDataSet_v0_User_v1.json&version=GBmaster. Please check its presence and validate in PROD runs first
2) Please add this field in:
a) "JSON representation" section 
b) add some sample entries/update existing rows from test output containing this field having some value